neofetch

```diff
@@ -8,7 +8,7 @@
 #
 # The MIT License (MIT)
 #
-# Copyright (c) 2015-2021 Dylan Araps
+# Copyright (c) 2015-2020 Dylan Araps
```

Comment:
Please make your changes on the latest neofetch script. You can do that by cloning the github repo.

Your current pull request is based on an older version of neofetch, so accepting it would revert other parts of neofetch back to 2020.